aspnet/hosting/aspnet-core-module.rst

```diff
@@ -0,0 +1,155 @@
+|                           | | %HTTP_PLATFORM_PORT%.                            |
+|                           | |                                                  |
+|                           | | The default value is "00:02:00".                 |
++---------------------------+----------------------------------------------------+
+| stdoutLogEnabled          | | Optional Boolean  attribute.                     |
+|                           | |                                                  |
+|                           | | If true, **stdout** and **stderr** for the       |
+|                           | | process specified in **processPath** will be     |
+|                           | | redirected to the file specified in              |
+|                           | | **stdoutLogFile**.                               |
+|                           | |                                                  |
+|                           | | The default value is false.                      |
++---------------------------+----------------------------------------------------+
+| stdoutLogFile             | | Optional string attribute.                       |
+|                           | |                                                  |
+|                           | | Specifies the relative or absolute file path for |
+|                           | | which **stdout** and **stderr** from the process |
+|                           | | specified in **processPath** will be logged.     |
+|                           | | Relative paths are relative to the root of the   |
+|                           | | site. Any path starting with '.' will be         |
+|                           | | relative to the site root and all other paths    |
+|                           | | will be treated as absolute paths.               |
+|                           | |                                                  |
+|                           | | The default value is ``httpplatform-stdout``.    |
++---------------------------+----------------------------------------------------+
+| forwardWindowsAuthToken   | | True or False.                                   |
+|                           | |                                                  |
+|                           | | If  true, the token will be forwarded to the     |
+|                           | | child process listening on %HTTP_PLATFORM_PORT%  |
+|                           | | as a header 'X-IIS-WindowsAuthToken' per request.|
+|                           | | It is the responsibility of that process to call |
+|                           | | CloseHandle on this token per request.           |
+|                           | |                                                  |
+|                           | | The default value is false.                      |
++---------------------------+----------------------------------------------------+
+
```

Comment:
`shutdownTimeLimit` was an accidental omission. I will update

`processesPerApplication` was intentionally left out. It is an artifact of forking form `httpPlatformHandler`. At the moment, we don't recommend you use it with ASP.NET Core applications